save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -2,21 +2,27 @@ package com.saveourtool.save.frontend.components.views.usersettings
 
 import com.saveourtool.save.domain.Role
 import com.saveourtool.save.frontend.components.basic.cardComponent
+import com.saveourtool.save.frontend.externals.fontawesome.faTrashAlt
+import com.saveourtool.save.frontend.externals.fontawesome.fontAwesomeIcon
+import com.saveourtool.save.frontend.utils.*
 import com.saveourtool.save.v1
-import csstype.ClassName
 
-import react.FC
+import csstype.ClassName
+import react.*
 import react.dom.html.ReactHTML.a
 import react.dom.html.ReactHTML.div
 import react.dom.html.ReactHTML.h1
 import react.dom.html.ReactHTML.img
 import react.dom.html.ReactHTML.li
 import react.dom.html.ReactHTML.ul
 
-@Suppress("MISSING_KDOC_TOP_LEVEL", "TOO_LONG_FUNCTION")
+val deleteOrganizationFun = deleteOrganizationButton()
```

Comment:
`*Fun` implies that the variable holds a method reference, but really it's a component

Reply:
Done